readme.md

```diff
@@ -1,135 +1,143 @@
->
->The following is the two columns of the pages excel.
->
->**Webbadress	Webbsida**
->
->The first is the url to be crawled.
->The second is a short description of the url that goes in the FGS node webbsida.
->
->load the excel file with a list of basmetadata for the  current run.
->
->**basmetadata** = pd.read_excel('**basmetadata_extern_webb.xlsx**', sheet_name='basmetadata')
->
->The following is the first column of the excel with basmetadata. The second column is for the values.
->
->**Basmetadata**
->
->Organisation
->
->Arkivbildare
->
->Arkivbildarenhet
->
->Arkiv
->
->Serie
->
->Klassificeringsstruktur
->
->nivå1
->
->nivå2
->
->nivå3
->
->Ursprung
->
->Sekretess
->
->Personuppgifter
->
->Forskningsdata
->
->Kommentar
- 
-# License
-This project is licensed under the GPL3 License. See the [LICENSE](LICENSE.txt) file for more information.  
-
-# Contributing
-Contributions are welcome! 
-
-
+# About
+> A python script to harvest screenshots from **websites and social media** and prepare them for archiving in the **E-archive LTA**.
+> 
+# Features
+>Takes screenshots of webpages or social media as **tiff** which is an archiveable image format.
+>
+>Creates XML with metadata to each screenshot in **FGS archive XML** format.
+>
+>The **FGS XML structure** is based on **FREDA which is an E-archive cooperation with municipalities in Jönköpings Län**.
+>
+>Some of the **metadata** in the **FGS XML** comes from an **excel file (Basmetadata)** and others are extracted from **the current crawled website**.
+>
+>Validates the FGS XML with .XSD file with the structure of **FREDA FGS XML for Websites**.
+>
+>Creates a file **Package-Creator-Metadata.xlsx** that is to be used by another script or process to build the final package with LTA:s software **Package Creator**.
+>
+# Installation
+
+
+# Prerequisites
+The application needs to be run on a Windows machine capable of running Python 3.
+ 
+
+#### Python Modules
+The following modules needs to be installed.
+ 
+Install them using pip by typing the following in the terminal.
+ 
+See https://pip.pypa.io/en/stable/installation/ for help installing pip.
+ 
+>python -m pip install Pillow
+>python -m pip install pandas
+>python -m pip install lxml
+>python -m pip install selenium
+>python -m pip install webdriver-manager
+>python -m pip install openpyxl
+>python -m pip pip install python-dotenv
+
+# Usage
+>Open **Spyder** (comes with **Anaconda**)
+>
+>Open the python file **archiving-of-web-and-social-media.py** in **Spyder**
+>
+>Set the data in the **config section** in the python file **archiving-of-web-and-social-media.py**
+>
+>There is already **sample data and a config setting** that works out of the box
+>but you need to put **your own settings** in the config section for your purpose.
+>See the configuration section in this readme for more information about the settings.
+>
+>Run the python script in **Spyder**
+>
```

Comment:
Detta kommer nog att behöva ändras sen, eller hur? När det är ett cli program får vi skriva in hur man kör programmet i terminalen och vilka flaggor, parametrar man kan/ska skicka med. Jag undrar om du ska ta bort en del av texten redan nu?